frontend/document/pdfdocument.lua

```diff
@@ -36,12 +33,11 @@ function PdfDocument:init()
     if not ok then
         error(self._document)  -- will contain error message
     end
+    self.reflowable_font_size = self:convertKoptToReflowableFontSize()
     -- no-op on PDF
-    self._document:layoutDocument(
-        CanvasContext:getWidth(),
-        CanvasContext:getHeight(),
-        CanvasContext:scaleBySize(self.epub_font_size))
+    self:layoutDocument()
     self.is_open = true
+    self.is_reflowable = self._document:isDocumentReflowable()
```

Comment:
nitpick: might move this one up so all the "reflowable" properties are close together

Reply:
True, it's available right after first opening the document.

Btw, not implemented here but `fz_make_bookmark` and `fz_lookup_bookmark` should be used before & after redoing the layout to return to the same location in the reflowable document.